Dockerfile

```diff
@@ -1,18 +1,14 @@
+ARG TAG="master"
 FROM python:3.7
 
-RUN python3 -m pip install requests flask gunicorn coverage 
+ENV FLASK_APP=art_to_ftp.py
+ENV APP_PORT=${APP_PORT:-5700}
 
-COPY . /local/art_to_ftp
-WORKDIR /local/art_to_ftp
+RUN python3 -m pip install flask gunicorn oc-cdtapi
```

Comment:
Please install only the package itself and move its dependencies to *setup.py*, as it specified in *Developers guide*: https://github.com/octopusden/octopus-base/blob/main/docs/Octopus%20Developer%20Guide.md